addons/point_of_sale/models/pos_order.py

```diff
@@ -1211,7 +1215,7 @@ def get_sale_details(self, date_start=False, date_stop=False, configs=False):
             ('state', 'in', ['paid','invoiced','done']),
             ('config_id', 'in', configs.ids)])
 
-        user_currency = self.env.company_id.currency_id
+        user_currency = configs.company_id.currency_id
```

Comment:
don't do that, it won't works because configs is more than one records so you (and the system) cannot guess the company if you have one config in company A and one in company B in variable config